packages/cli/src/lib/cmds/play.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {
+  Config, GradleWrapper, JdkHelper, AndroidSdkTools, ConsoleLog, Log, GooglePlay, TwaManifest,
+} from '@bubblewrap/core';
+import {Track} from '@bubblewrap/core/dist/lib/GooglePlay';
+import * as fs from 'fs';
+import * as path from 'path';
+import {ParsedArgs} from 'minimist';
+import {TWA_MANIFEST_FILE_NAME} from '../constants';
+import {Prompt, InquirerPrompt} from '../Prompt';
+/**
+  * The Play class is the class that is used to communicate with the Google Play Store.
+  */
+class Play {
+  constructor(
+    private args: ParsedArgs,
+    private config: Config,
+    private androidSdkTools: AndroidSdkTools,
+    private googlePlay: GooglePlay,
+  ) {}
+
+  async bootstrapPlay(): Promise<void> {
+    await this.googlePlay.initPlay();
+  }
+
+  // bubblewrap play --versionCheck can validate the largest version number vs twa-manifest.json and update to give x+1 version number.
+  async getLargestVersion(): Promise<Number> {
+    // Need to get an editId, then list all apks available. This should allow us to query the highest apk number.
+    // This exists in Gradle play plugin but is not easily accessible over CLI.
+    // This should be completed in a future CL.
+    return 0;
+  }
+
+  private isInAvailableTracks(userSpecifiedTrack: string): boolean {
+    const track: string = userSpecifiedTrack.toLowerCase();
+    const selectedTrack: Track = (Track as any)[track];
```

Comment:
So I think you could change this to `Track[track as keyof typeof Track]` (see the comments here [1]), which at least to me makes things seem a little clearer.

[1]: https://stackoverflow.com/a/17381004

Reply:
We could implement `Track` [similar to how we implement `Orientation`](https://github.com/GoogleChromeLabs/bubblewrap/blob/main/packages/core/src/lib/TwaManifest.ts#L49-L61) and get rid of this cast.

Reply:
I am starting down this path.

Reply:
I think I got this resolved.